.eslintrc.js

```diff
@@ -42,24 +43,96 @@ module.exports = {
     "es6": true
   },
   rules: {
+    "node/no-unsupported-features/es-syntax": "off",
```

Comment:
I moved this up from the overrides below. Since this rule applies to the entire codebase, it didn't make sense to me to have it expressed as an override.